libcryptsetup-rs-sys/build.rs

```diff
@@ -24,7 +27,16 @@ fn build_safe_free() {
 }
 
 fn generate_bindings(safe_free_is_needed: bool) {
-    let builder = bindgen::Builder::default().header("header.h").size_t_is_usize(true);
+    match pkg_config::probe_library("libcryptsetup") {
+        Ok(l) => {
+            let builder = bindgen::Builder::default()
+                .clang_args(l.include_paths.iter().map(|path| {
+                    let r = format!("-I{}", path.to_string_lossy());
+                    eprintln!("{}", r);
+                    r
+                }))
+                .header("header.h")
+                .size_t_is_usize(true);
     #[cfg(target_arch = "x86")]
     let builder = builder.blocklist_type("max_align_t");
     let builder_with_safe_free = if safe_free_is_needed {
```

Comment:
Could you fix the formatting for this block? It makes it a little bit hard to read but doesn't seem to be caught by cargo fmt because of the macro.